### ISSUE TITLE
[Bug]: Response can't be cached

### ISSUE THREAD

**Issue by paulhaem** [pimcore/pimcore#16057]
### Pimcore version

11.0.11

### Steps to reproduce

1. Activate Full Page Cache ([docs](https://pimcore.com/docs/platform/Pimcore/Development_Tools_and_Details/Cache/Full_Page_Cache#configure-the-full-page-cache))
3. Open any page on https://demo.pimcore.com/ as a non-authenticated user
4. Open the network tab in the developer console
5. Refresh the page
6. Inspect the response headers

### Actual Behavior

```
X-Pimcore-Output-Cache-Disable-Reason: Response can't be cached
```

Check happens here: https://github.com/pimcore/pimcore/blob/11.x/bundles/CoreBundle/src/EventListener/Frontend/FullPageCacheListener.php#L306

### Expected Behavior

Something like this should be outputed to indicate that the page is coming from the cache:
```
X-Pimcore-Cache-Date: 2011-10-23T00:55:37+02:00
X-Pimcore-Cache-Tag: output_cf61488e7b6c187f465923d8dcb2a8dd
```

**Comment by blankse** [comment]
I can confirm this bug. I think it is a regression of https://github.com/pimcore/pimcore/pull/14581

**Comment by blankse** [comment]
If you create a new Response object, the default value is "no-cache, private" (which is a reason to diable full page cache now).
So Pimcore and every user, which use e. g. `$this->render()` in the controllers, must set cache control headers to activate full page cache again.

Was this intended by the pimcore core team? FYI @dvesh3

**Comment by piotrecky** [comment]
You can use the event: FullPageCacheEvents::PREPARE_RESPONSE and check the appropriate conditions yourself and determine whether the response can be cached.

https://github.com/pimcore/pimcore/blob/11.x/lib/Event/FullPageCacheEvents.php#L50

**Comment by brusch** [comment]
Maybe we should remove `no-cache` and `private` then from the list here: 
https://github.com/pimcore/pimcore/blob/68f198d32add713f2e4025b5dcb6414754133145/bundles/CoreBundle/src/EventListener/Frontend/FullPageCacheListener.php#L366

And just use `no-store` - WDYT?

**Comment by blankse** [comment]
@brusch That was also my thought.
I created a PR: https://github.com/pimcore/pimcore/pull/16171

**Comment by brusch** [comment]
@blankse nice, thanks a lot 👍😊

**Comment by dvesh3** [comment]
Fixed by #16171
